src/themes/uclouvain/styles/_global-styles.scss

```diff
@@ -4,7 +4,7 @@
 @import '../../../styles/_global-styles.scss';
 
 .facet-filter, .setting-option {
-  background-color: var(--bs-light);
+  background-color: rgba(0, 0, 0, 0.01);
```

Comment:
opacity isn't a good idea for a background color, use "bs-gray-xxx" or define a custom background color variable